papr/README.md

```diff
@@ -10,12 +10,22 @@ enhancements:
 - the GHPRB plugin is installed by default
 - anonymous users have read access
 - a GitHub token is securely installed at runtime using OpenShift secrets
+- credentials for Jenkins Job Builder reconfiguration
 
 The `jenkins/` directory is used by the S2I
 [OpenShift Jenkins builder](https://github.com/openshift/jenkins/tree/8e58d88#installing-using-s2i-build)
-to create the customized Jenkins image.
+to create the customized Jenkins master image.
 
-# Usage
+The `slave/` directory is used by the `papr-slave` buildconfig to create an
+image for our Jenkins slave agent. This is used with the
+[Kubernetes plugin](https://plugins.jenkins.io/kubernetes).
```

Comment:
Are we using the image from Hub? https://hub.docker.com/r/jenkins/jnlp-slave/

Reply:
We're using the official [OpenShift jslave image](https://hub.docker.com/r/openshift/jenkins-slave-base-centos7/). It adds some things to help integrate it into Jenkins, e.g. resource control passing to the JVM: https://github.com/openshift/jenkins/tree/master/slave-base.